localization/react-intl/src/app/components/team/Statuses/DeleteStatusDialogBody.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "id": "deleteStatusDialogBody.statusInUseTitle",
+    "defaultMessage": "You need change the status of 76 items to delete this status"
```

Comment:
Missing word ("you need *to* change") and hard-coded value ("76") instead of placeholder.